scripts/install-systemd-multi-user.sh

```diff
@@ -41,10 +41,8 @@ handle_network_proxy() {
     fi
 }
 
-poly_validate_assumptions() {
-    if [ "$(uname -s)" != "Linux" ]; then
-        failure "This script is for use with Linux!"
-    fi
+poly_cure_artifacts() {
+    :
```

Comment:
Why not remove this completely? It does not do anything right now.

Reply:
This pattern is used in the `install-{platform}-multi-user.sh` scripts for a function that the outer `install-multi-user.sh` script calls, but which some platforms don't need to invoke.

They're just explicit no-ops to avoid the command-not-found error and be explicit about the API.